pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -690,7 +676,6 @@ func (b *BridgeBindMechanism) switchPodInterfaceWithDummy() error {
 
 type MasqueradeBindMechanism struct {
 	vmi                 *v1.VirtualMachineInstance
-	dhcpConfig          *cache.DhcpConfig
 	iface               *v1.Interface
```

Comment:
Not related to this PR, just for the TODO list. Can you please change this name to - `vmiSpecInterface`. Same for the `iface` in the`podNIC`.

Reply:
Written down; resolving.